wandb/hashutil.py

```diff
@@ -0,0 +1,47 @@
+import base64
```

Comment:
mind moving it under lib for now (just don't want util files scattered in top level)